card-set/src/card-component.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import './Card.css';
+
+const UserCard = (props) => {
+  const [avatar, setAvatar] = React.useState('https://media.tenor.com/images/e549f9798674301c5af2c91581194091/tenor.gif')
```

Comment:
👍 👍 - this is a cool solution for the default state of the avatar. Nice thinking!

Reply:
I probably would have used `null` as the default value for all of these pieces of state, and then conditionally rendered a 'Loading' message. 

```jsx
// example
const [firstName, setFirstName] = useState(null);

...

if(!firstName){
  return <p>...Loading</p>
}

//...rest of the card
```

But having the default loading image makes this work kind of well, too!